android/assets/jsons/Nations/Nations_Italian.json

```diff
@@ -182,7 +189,8 @@
 		"unique": "+2 Cultura al turno dalle città prima della scoperta dell'Energia a Vapore",
 		"cities": ["Parigi","Orleans","Lione","Troyes","Tours","Marsiglia","Chartres","Avignon","Rouen","Grenoble",
 			"Dijon","Amiens","Cherbourg","Poitiers","Tolosa","Bayonne","Strasburgo","Brest","Bordeaux","Rennes",
-			"Nizza","Saint Etienne","Nantes","Reims","Le Mans","Montpellier","Limoges","Nancy","Lilla","Caen"]
+			"Nizza","Saint Etienne","Nantes","Reims","Le Mans","Montpellier","Limoges","Nancy","Lilla","Caen"]				"Nizza","Saint Etienne","Nantes","Reims","Le Mans","Montpellier","Limoges","Nancy","Lilla","Caen","Toulon",
```

Comment:
This line. There is an extra ] after Caen

Reply:
Okay, corrected